orchagent/routeorch.cpp

```diff
@@ -4,6 +4,7 @@
 #include "routeorch.h"
 #include "logger.h"
 #include "swssnet.h"
+#include "converter.h"
```

Comment:
what is this header file for? if not used, please remove

Reply:
@prsunny Done.  It is removed.  Calls that used it were moved to mplsrouteorch.cpp.